src/org/thoughtcrime/securesms/PinnedMessageAdapter.java

```diff
@@ -120,9 +120,20 @@ public void onClick(View v) {
     }
 
     private void setMessageView(MessageRecord record, ViewHolder viewHolder) {
-        if(!record.isOutgoing()) {
-            viewHolder.recipient.setText(record.getRecipient().getName());
+        String ownName = "Me";
```

Comment:
This should be defined in strings.xml and referenced here. See https://developer.android.com/guide/topics/resources/string-resource.html